tests/fixtures/project_with_requirements_in/requirements.txt

```diff
@@ -16,3 +16,5 @@ urllib3==2.2.3
     # via -r requirements.in
 uvicorn==0.32.0
     # via -r requirements.in
+
+beautifulsoup4
```

Comment:
Could you add `beautifulsoup4==4.12.3` [here](https://github.com/fpgmaas/deptry/blob/df666f768532f5d988be2109c7625df882b2d5fa/tests/fixtures/project_with_requirements_in/requirements.in#L1), and run `uvx --from pip-tools pip-compile` from `tests/fixtures/project_with_requirements_in` instead, to make sure it's also added to the source file?